### DISCUSSION TITLE
Shall we plan a meeting today or next week?

### DISCUSSION THREAD

**Discussion by Brbauwens** [vkmbox/Ml-readinggroup#4] [General]
1) Yesterday evening I returned to the Roberts and Yaida book. Remember that I was currently working on a concise and rigorous way to calculate 4th moments of preactivations, and was writing a note. 

There are still mathematical problems, so, I am not ready to explain something today. I am hopeful that I may be ready by next week...

2) Aleksandr has been reading the paper [On lazy training in differentiable programming](https://proceedings.neurips.cc/paper/8559-on-lazy-training-in-differentiable-programming). It is a nicely written note that stresses the importance of scaling. But he/we do not yet understand the paper yet.

==> Shall we postpone the meeting to next week? Or are there other things we should discuss. For example, Viacheslav, if you feel stuck with your experiments, then let's discuss them.

**Comment by vkmbox** [comment]
Currently I plan to add experiments with tangent activation at criticality; also, on output layer there is no-zero covariation between neurons which must be explained. I think we may skip this time to accumulate materials to discuss.

**Reply by vkmbox** [reply]
@Brbauwens pls let me know if we are finally planning the meeting this week

**Reply by Brbauwens** [reply]
Ok, let us skipp the meeting today so we can accumulate results. Let us plan a meeting next week at 05.08 at the usual hour anyways, provided this works for everyone.

**Reply by Brbauwens** [reply]
PS: to find the mistake in the calculation of covariation, maybe you can start by checking the 1st hidden layer, where preactivations are always Guassian, and apply a statistical test for covariation of 2 Gaussian distributions. Maybe there exists such a test for xi-squared distribution? Then you can try the 2nd layer of the linear network...

**Reply by Brbauwens** [reply]
Correction: it is not chi-squared, but this distribution https://math.stackexchange.com/questions/484300/sums-of-products-of-two-normal-variables
